paasta_tools/docker_wrapper.py

```diff
@@ -341,6 +343,10 @@ def main(argv=None):
 
     if mesos_task_id and can_add_hostname(argv):
         hostname = generate_hostname(socket.getfqdn(), mesos_task_id)
+        # Add PAASTA_HOST environment variable
```

Comment:
Due to the complexity of how this works, and the fact that it is implemented so that it works only with the wrapper (and won't work in k8s), I vote to defer on `PAASTA_HOST` for now.

I like the other variables though!

Reply:
Disregard, I didn't see the k8s part.